src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+@import './utils/variables';
+@import './blocks/calendar';
+
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+
+  &__day {
+    box-sizing: border-box;
+    list-style: none;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      animation: move 0.5s;
+    }
+  }
+
+  @for $number from 1 through 31 {
+    & :nth-child(#{$number})::before {
+      content: '#{$number}';
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+    }
+  }
+
+  $days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+
+  @each $day, $dayCount in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: 101px * $dayCount;
+    }
+  }
+
+  @for $number from 28 through 31 {
+    &--month-length-#{$number} :nth-child(n + #{$number + 1}) {
+      display: none;
+    }
+  }
+
+  @keyframes move {
+    from {
+      background-color: #eee;
+      transform: translate(0);
+    }
+
+    to {
+      background-color: #ffbfcb;
+      transform: translate(0, -20px);
+    }
+  }
```

Comment:
The styles for the '.calendar' class and its children are duplicated from 'calendar.scss'. To avoid redundancy and potential inconsistencies, consider removing these styles from this file and relying solely on the imported 'calendar.scss'.